httpie/cli/argtemplate.py

```diff
@@ -95,6 +99,12 @@ def edit_json_template(args):
 
 
 def load_template(arg):
+    """
+    Load a template from templates.json and return a list of arguments to be passed to the main function
+    the format for running this is:
+        http <name> <item> <value>
```

Comment:
I think this is wrong, to run a template you just do `http runt <template>`

Reply:
Thanks @editf, I'll fix it!